translations/fr/pack/side/mtt_encounter.json

```diff
@@ -0,0 +1,67 @@
+[
+    {
+        "code": "87014",
+        "cost": 1,
+        "deck_limit": 1,
+        "encounter_code": "machinations_through_time",
+        "encounter_position": 13,
+        "faction_code": "neutral",
```

Comment:
In translated jsons, you don't need to keep the whole card metadata, just `code` + the translated fields. In this case, the English source file to translate would be:
```
[
        {
                "code": "87014",
                "flavor": "The high-voltage demonstrations performed by Tesla throughout America and Europe were truly awe-inspiring.",
                "name": "Nikola Tesla",
                "slot": "Ally",
                "subname": "Renowowned Inventor",
                "text": "[fast] Exhaust Nikola Tesla: <b>Parley.</b> Test [intellect] (1). If you succeed, either gain 1 clue <i>(from the token pool)</i>, draw 2 cards, or gain 3 resources. (Max once per game per option.)",
                "traits": "Scientist. Ally. Past."
        },
        {
                "code": "87023",
                "name": "Ezra Graves",
                "slot": "Ally",
                "subname": "Professor of the Arcane",
                "text": "[fast] During any investigator's turn, exhaust Ezra Graves and spend 1 resource: Immediately take an action as if it were your turn.\n[action]: Choose an [[Ally]] asset in your discard pile and put it into play under your control (Max once per game.)",
                "traits": "Scientist. Ally. Present."
        },
        {
                "code": "87032",
                "name": "Dimensional Beam Machine",
                "text": "Uses (2 charges).\n[fast] During your turn, exhaust Dimensional Beam Machine and spend 1 charge: Move an investigator or a non-[[Elite]] enemy at your location to any revealed location.",
                "traits": "Item. Science. Tool. Future."
        }
]
```